src/triggers/project.ts

```diff
@@ -39,12 +42,15 @@ const getProjectList = async (z: ZObject, bundle: Bundle) => {
             first: 50
             after: $after
             orderBy: updatedAt
-            filter: { state: { in: ["backlog", "started", "planned", "paused"] } }
+            filter: { status: { type: { in: ["backlog", "started", "planned"] } } }
```

Comment:
Why removing paused?

Reply:
Will no longer be a valid `state` after LIN-19693